homeassistant/components/media_player/ue_smart_radio.py

```diff
@@ -46,7 +46,7 @@ def send_request(payload, session):
     try:
         request = requests.post(
             URL, cookies={"sdi_squeezenetwork_session": session},
-            json=payload, timeout=5)
+            json=payload, timeout=30)
```

Comment:
Is this URL also slow? (it is not www.uesmartradio.com)

Since this is called from `update()`, we should probably increase `SCAN_INTERVAL` if we change this timeout.

Reply:
@amelchio @pvizeli 

Thanks for the feedback. I did a few tests and `http://decibel.logitechmusic.com/jsonrpc.js` may indeed not be as slow as `www.uesmartradio.com`. I do not know what `SCAN_INTERVAL` is, though (and therefore the implications of changing the timeout).

I also noticed that the `requests` call in `setup_platform` is not in a `try:`, which may have border effects once `www.uesmartradio.com` times out. I can make a PR for that, I would just need to understand what should happen in `setup_platform` when there is a communication problem (nothing maybe?)

Reply:
You set this on top of the platform like `SCAN_INTERVAL=timedelta(seconds=60)`

Reply:
@pvizeli This is polling a website, do you think 10s SCAN_INTERVAL is even reasonable?

I think we should raise the SCAN_INTERVAL and lower the timeout :)

Reply:
Yeah hitting a website every 10 seconds is brutal.